disorder-jack/src/Disorder/Jack/Combinators.hs

```diff
@@ -122,6 +123,12 @@ sizedIntegral :: Integral a => Jack a
 sizedIntegral =
   mkJack QC.shrinkIntegral QC.arbitrarySizedIntegral
 
+-- | Generates a realfrac number. The number can be positive or negative and
+--   its maximum absolute value depends on the size parameter.
+sizedFloat :: Savage.RealFrac a => Jack a
+sizedFloat =
+  mkJack QC.shrinkRealFrac QC.arbitrarySizedFractional
```

Comment:
Would prefer if this was called `sizedRealFrac` as that's what the type class is called.

No need to import it as `Savage`, the only reason it isn't available already is that Jack doesn't depend on P. You can happily add an `import Prelude (RealFrac)`.